tests/test_torch_nn.py

```diff
@@ -350,13 +350,29 @@ def test_torch_nn_functional_prelu():
   t2j_function_test(torch.nn.functional.prelu, [(5, 3, 112, 122), (3,)], atol=1e-6)
 
 
+def test_torch_nn_functional_silu():
+  t2j_function_test(torch.nn.functional.silu, [(6, 6)], atol=1e-6)
+  t2j_function_test(torch.nn.functional.silu, [(5, 3, 112, 122)], atol=1e-6)
+
+
 def test_torch_nn_functional_scaled_dot_product_attention():
   t2j_function_test(lambda x, y: x @ y, [(2, 3, 5), (5, 7)], atol=1e-6)
 
   sdpa = torch.nn.functional.scaled_dot_product_attention
+
+  # default
   t2j_function_test(sdpa, [(5, 3, 7), (5, 2, 7), (5, 2, 7)], atol=1e-6)
   t2j_function_test(sdpa, [(5, 7, 11), (5, 7, 11), (5, 7, 11)], atol=1e-6)
 
+  # default + attn_mask
```

Comment:
can you please also add a test for the case when `attn_mask` has dtype bool? i think that should cover all branches